server/utils/fileUtils.test.ts

```diff
@@ -0,0 +1,9 @@
+import FileUtils from './fileUtils'
+
+describe(FileUtils, () => {
+  describe('.fileInformation', () => {
+    it('returns the file type and size in KB, rounded to one decimal place', async () => {
```

Comment:
I reckon we can return this in bytes; converting to KB and rounding can probs be considered a formatting concern.